dali/test/python/test_torch_pipeline_rnnt.py

```diff
@@ -159,7 +159,9 @@ def forward(self, inp, seq_len):
         # do stft
         x = torch.stft(x, n_fft=self.n_fft, hop_length=self.hop_length,
                        win_length=self.win_length, pad_mode='reflect',
-                       center=True, window=self.window.to(dtype=torch.float).to(x.device))
+                       center=True, window=self.window.to(dtype=torch.float).to(x.device),
+                       return_complex=True)
+        x = torch.view_as_real(x)
```

Comment:
Why do we need this change?

Reply:
According to https://pytorch.org/docs/stable/generated/torch.stft.html:
```
From version 1.8.0, return_complex must always be given explicitly for real inputs and return_complex=False has been deprecated. Strongly prefer return_complex=True as in a future pytorch release, this function will only return complex tensors.

Note that [torch.view_as_real()](https://pytorch.org/docs/stable/generated/torch.view_as_real.html#torch.view_as_real) can be used to recover a real tensor with an extra last dimension for real and imaginary components.
```